scylla-cql/src/utils/mod.rs

```diff
@@ -1 +1,4 @@
+//! Various utilities.
+// TODO(2.0): Deconstruct this module by extracting submodules to the top level.
+
```

Comment:
Why? And what modules? There is only parse here.

Reply:
Exactly. I fail to see how the `parse` module is an "util" more than any other module. That's why I suggest getting read of the `utils` module altogether.